Source/WebCore/dom/TreeScope.cpp

```diff
@@ -593,29 +594,38 @@ SVGResourcesMap& TreeScope::svgResourcesMap() const
     return *m_svgResourcesMap;
 }
 
+void TreeScope::addSVGResource(const AtomString& id, RenderSVGResourceContainer& )
+{
+    if (id.isEmpty())
+        return;
+
+    // Replaces resource if already present, to handle potential id changes
+    // svgResourcesMap().resources.set(id, &resource);
```

Comment:
Why is it commented out? :-)

Reply:
Does not seem needed to comment out indeed, reinstated.